src/components/account/list/UserRow.vue

```diff
@@ -38,7 +38,7 @@
                 <div class="flex-col items-baseline" :class="[isStudent ? 'flex' : 'hidden']">
                     <div class="leading-5 text-blue-900 ml-1 mb-1">{{ student.matriculationId }}</div>
                     <div class="hidden sm:flex items-center leading-5 text-gray-500">
-                        <span class="mr-2 fa-stack text-xs" style="font-size: 0.63em">
+                        <span class="mr-2 fa-stack text-xs" style="font-size: 0.63em;">
```

Comment:
```suggestion
                        <span class="mr-2 fa-stack text-xs" style="font-size: 0.63em">
```

is this semicolon intended? Just dismiss this suggestion if it is